cmd/global-heal.go

```diff
@@ -143,16 +143,6 @@ func (er *erasureObjects) healErasureSet(ctx context.Context, buckets []string,
 	healBuckets := make([]string, len(buckets))
 	copy(healBuckets, buckets)
 
-	// Heal all buckets first in this erasure set - this is useful
-	// for new objects upload in different buckets to be successful
-	for _, bucket := range healBuckets {
```

Comment:
As bucket healing is now moved at node level, trigger of bucket healing using erasure sets not needed. Also as part of `globalBucketMetadataSys.Init` the bucket healing at node level would be triggered. While healing the erasure sets, we heal the objects.
My assumption may be wrong and if so kindly let me know.

Reply:
Do you mean something like
```
	for _, bucket := range healBuckets {
		_, err := globalBucketMetadataSys.objAPI.HealBucket(ctx, bucket, madmin.HealOpts{ScanMode: scanMode})
		if err != nil {
			// Log bucket healing error if any, we shall retry again.
			logger.LogIf(ctx, err)
		}
	}
```
should be used?

Reply:
I would suggest using `objAPI` directly instead of accessing a private variable of a global method.

You can create an API like this though 

```
globalBucketMetadataSys.HealBucket()
```

Reply:
Done